website/promotion/emails.py

```diff
@@ -1,16 +1,55 @@
 """The emails defined by the promotion request package."""
+import logging
+from smtplib import SMTPException
+
 from django.conf import settings
 from django.utils.translation import gettext_lazy as _
+from django.utils import translation, timezone
+from django.template.loader import get_template
+from django.core import mail
+from django.core.mail import EmailMultiAlternatives
+
+from promotion import services
+
+logger = logging.getLogger(__name__)
+
+def send_weekly_overview():
+
+    new_requests = services.get_new_requests(timezone.localtime())
+    upcoming_requests = services.get_upcoming_requests(timezone.localdate())
+
+    from_email = settings.PROMO_REQUEST_NOTIFICATION_ADDRESS
+    text_template = get_template("requests/weekly_overview.txt")
```

Comment:
please use utils.send_email for sending emails and don't try things like setting language and translations or catching SMTP errors yourself in here.